pkg/mgrconfig/mgrconfig.go

```diff
@@ -85,9 +85,10 @@ type Config struct {
 
 	// Implementation details beyond this point.
 	// Parsed Target:
-	TargetOS     string `json:"-"`
-	TargetArch   string `json:"-"`
-	TargetVMArch string `json:"-"`
+	TargetOS           string `json:"-"`
+	TargetArch         string `json:"-"`
+	TargetVMArch       string `json:"-"`
+	TargetKernelObject string `json:"-"`
```

Comment:
Let's not expose it at all. Adding user options is easy. It's removing them later that's hard.

Manager type has target field. Add sysTarget field next to it and populate when manager is created and then use mgr.sysTarget.KernelObject where needed.
If there will be actual need to make it configurable, then we can expose it then. But I would prefer later (or never) than sooner.

Reply:
Fixed